src/hooks/v2/transactor/ClaimTokensTx.ts

```diff
@@ -15,6 +15,22 @@ export function useClaimTokensTx(): TransactorInstance<{
 
   return ({ claimAmount }, txOpts) => {
     if (!transactor || !userAddress || !projectId || !contracts?.JBTokenStore) {
+      const missingParam = !transactor
+        ? 'transactor'
+        : !userAddress
+        ? 'userAddress'
+        : !projectId
+        ? 'projectId'
+        : !contracts?.JBTokenStore
+        ? 'contracts.JBTokenStore'
+        : null
+
+      txOpts?.onError?.(
+        new DOMException(
+          `Missing ${missingParam ?? 'parameter not found'} in v2 transactor`,
+        ),
+      )
```

Comment:
@NakedFool one final thing: could you please update these to use the new-ish `onCatch` function?

https://github.com/jbx-protocol/juice-interface/blob/4a593744b502a7c8b3debcf6a844e6a149986a83/src/hooks/Transactor.ts#L32

Reply:
sure! 🍻